tests/testcases/015_check-nodes-ready.yml

```diff
@@ -1,18 +1,10 @@
 ---
 - name: Testcases checking nodes
   hosts: kube_control_plane[0]
+  vars:
+    bin_dir: /usr/local/bin
   tasks:
 
-  - name: Force binaries directory for Flatcar Container Linux by Kinvolk
-    set_fact:
-      bin_dir: "/opt/bin"
-    when: ansible_os_family in ["Flatcar", "Flatcar Container Linux by Kinvolk"]
-
-  - name: Force binaries directory for other hosts
-    set_fact:
-      bin_dir: "/usr/local/bin"
-    when: not ansible_os_family in ["Flatcar", "Flatcar Container Linux by Kinvolk"]
-
```

Comment:
I think the removal of the Flatcar test case is acceptable, although it's somewhat regrettable. It would be great if we could add the Flatcar test case in the future.

Reply:
https://github.com/kubernetes-sigs/kubespray/issues/11804

Reply:
Yeah I don't mind adding a test ; but in the meantime, we should not adjust for things we don't actually test.
This isn't hard to revert later if needed.